experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -188,6 +239,39 @@ export const makeFieldEditGenerator = (
 		return { type: "sequence", edit: contents };
 	}
 
+	function generateValueFieldDeleteOp(fieldPath: FieldUpPath, treeIndex: number): ValueFieldEdit {
```

Comment:
It's probably worth factoring out a `generateDeleteEdit()` function from this function and the function below

Reply:
factored out generateDeleteEdit() function